x-pack/plugins/maps/public/selectors/map_selectors.ts

```diff
@@ -241,6 +244,7 @@ export const getDataFilters = createSelector(
   getSearchSessionId,
   getSearchSessionMapBuffer,
   getIsReadOnly,
+  getEditModeActiveForLayer,
```

Comment:
getEditModeActiveForLayer can be removed, its never used. Make sure to clean up the tests